cmd/oras/root/pull.go

```diff
@@ -70,6 +71,9 @@ Example - Pull artifact files from a registry:
 Example - Recursively pulling all files from a registry, including subjects of hello:v1:
   oras pull --include-subject localhost:5000/hello:v1
 
+Example - Pull files from a directory-based artifact in a registry, preserving original permissions
+oras pull --preserve-permissions -o out localhost:5000/hello-dir:v1
```

Comment:
nit: formatting. Also, remove the non-relevant flag `-o`.
```suggestion
  oras pull --preserve-permissions localhost:5000/hello-dir:v1
```